packages/react/src/avatar/Avatar.css.ts

```diff
@@ -18,10 +18,10 @@ export const avatar = recipe({
 
       selectors: {
         [`${styles.className} &:not(:first-child)`]: {
-          marginLeft: "-12px",
+          marginLeft: "-3px",
         },
         [`${styles.className} &`]: {
-          border: `2px solid ${theme.colors["white"]}`,
+          outline: `1px solid ${theme.colors["white"]}`,
```

Comment:
let's switch back to border here since we want the element to fit within the given size - outline will spill the content outside the bounding box.